### PR TITLE
First pass, end-to-end tests

### PR DESCRIPTION
**Pull request by francisli**

### CODE REVIEW

**Comment by francisli** [comment]
@fwextensions what do you think? any changes you'd like to suggest before merging it back onto your original branch and working together there?

File: e2e/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "e2e",
+  "version": "1.0.0",
```
Comment (fwextensions) [inline e2e/package.json:3]: Doesn't really matter, but the other packages are at 0.1.0.  Not sure if there's any versioning strategy needed.

File: example.env [hunk 2]
```diff
@@ -8,3 +8,8 @@ PR_CLIENT_SECRET=replaceme
 REACT_APP_DISABLE_CODE_3=
 REACT_APP_DISABLE_PILOT_HOSPITALS=
 SESSION_SECRET=makeitasecretinprod
+
+EMS_USER=op.ems@c4sf.me
+EMS_PASS=abcd1234
```
Comment (fwextensions) [inline example.env:13]: Do we want these passwords in the repo?

**Review by fwextensions — APPROVED** [review]
Looks good.  Does the Circle CI image change address #187 ?

File: example.env (hunk 2 above)
Comment (francisli) [reply example.env:13]: I think these dummy seed accounts/passwords are fine, they're obviously placeholder...

File: e2e/package.json (hunk 1 above)
Comment (francisli) [reply e2e/package.json:3]: I'll make it match the others... we should think about versioning...

**Comment by francisli** [comment]
> Looks good. Does the Circle CI image change address #187 ?

Yes, I believe it does

**Pull request merged**